### PR TITLE
[stable-1.1.x] Bump chart to 0.8.43

### PR DESCRIPTION
**Pull request by gracedo**
**What type of PR is this?**
Feature

**What this PR does/ why we need it**:
Bump kommander to 0.8.43 patch version (https://github.com/mesosphere/charts/pull/734) to pull in kubecost upgrades. Once this is merged, I will cut 1.1.1 tags.

**Which issue(s) this PR fixes**:

D2IQ-70685
**Special notes for your reviewer**:

I tested this on a cluster and verified that the thanos query offset was set to 5m and costs came up shortly after the cluster was attached.

Depends on https://github.com/mesosphere/charts/pull/736

**Does this PR introduce a user-facing change?**:

```release-note
kommander: Aggregated cost monitoring data is now available 5 minutes after clusters are attached
```

**Checklist**

- [x] The commit message explains the changes and why are needed.
- [x] The code builds and passes lint/style checks locally.
- [ ] The relevant subset of integration tests pass locally.
- [ ] The core changes are covered by tests.
- [ ] The documentation is updated where needed.

### CODE REVIEW

File: addons/kommander/1.1.1/kommander-1.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kommander
+  labels:
+    kubeaddons.mesosphere.io/name: kommander
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.1.-1"
```
Comment (samvantran) [inline addons/kommander/1.1.1/kommander-1.yaml:9]:
```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "1.1.1-1"
```

**Comment by samvantran** [comment]
Will you forwardport this to master after?

**Comment by gracedo** [comment]
I'll merge this revision into master, but the feature itself is on the most recent version of the chart (0.9.4), and bumped in this open pr https://github.com/mesosphere/kubeaddons-kommander/pull/132 . It's a little confusing because this same feature exists in two different versions of the chart - this patch 0.8.43 and latest 0.9.4 chart versions, but they'll be going into kommander 1.1.1 and future 1.2.0 respectively

**Comment by samvantran** [comment]
Gotcha. It's fine if the feature exists in two different versions of the chart. I was asking because it would be good to have the  revision added and make the complete full list of versions in https://github.com/mesosphere/kubeaddons-kommander/tree/master/addons/kommander (otherwise master will be missing the 1.1.1 folder)

**Comment by makkes** [comment]
CI is supposedly failing because the kubeaddons deploy that is in use in CI enables the Helm 3 driver which the Kommander Addon is not compatible with.

**Comment by gracedo** [comment]
Since the change included in this patch is fairly minor (kubecost upgrade) we've agreed to merge and cut the 1.1.1 tags now to unblock konvoy 1.5.1. Tests will be run in Konvoy so that should catch anything just in case.

**Pull request merged**